### ISSUE TITLE
🐛 BUG: Incorrect package name for solidjs renderer when upgrading to 0.25

### ISSUE THREAD

**Issue by zadeviggers** [withastro/astro#2910]
### What version of `astro` are you using?

0.25.2

### What package manager are you using?

pnpm

### What operating system are you using?

Windows

### Describe the Bug

When upgrading a project that used solidJS components to 0.25.2, the incorrect package name is suggested in the `astro add` command.

For example, with this config file:
```js
export default {
	renderers: ["@astrojs/renderer-solid"],
};
```
Running `astro add` yeilds the following message:
```
Astro "renderers" are now "integrations"!
Update your configuration and install new dependencies:

  // astro.config.js
  import solid from '@astrojs/solid';

  // ...
  integrations: [
    solid(),
  ],
```
This is incorrect, because the name of the integration for solid is `@astrojs/solid-js`, not `@astrojs/solid`.

Running `pnpm i @astrojs/solid` yeilds this message:
```
@astrojs/solid is not in the npm registry, or you have no permission to fetch it.
```

### Link to Minimal Reproducible Example

N/A

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by RafidMuhymin** [comment]
#2911 should fix this

**Comment by tony-sull** [comment]
Fixed in #2911 🥳
